Watchman.Discord/Areas/Protection/Controllers/MuteUserController.cs

```diff
@@ -30,27 +39,28 @@ public MuteUserController(MutingService mutingService, UnmutingService unmutingS
             this._mutingHelper = mutingHelper;
         }
 
-        [DiscordCommand("mute")]
         [AdminCommand]
-        public async Task MuteUser(DiscordRequest request, Contexts contexts)
+        public async Task MuteUser(MuteCommand command, Contexts contexts)
         {
-            var requestParser = new MuteRequestParser(request, this._usersService, contexts);
-            var userToMute = requestParser.GetUser();
-            if (userToMute.Id == this._usersService.GetBot().Id)
+            var userToMute = await this._usersService.GetUserByIdAsync(contexts.Server, command.User);
+            if (userToMute == null)
             {
```

Comment:
a co gdybyśmy chcieli zmutować na przyszłość kogoś kto wyszedł zaraz po zrobieniu czegoś?

Reply:
czyli, że jak ma być?

Reply:
na razie to zostawmy, zrobi się kiedy indziej bo wymaga to lekkiej zmiany MuteService, a lepiej już jak najszybciej skończyć ten task